imap_processing/tests/lo/test_lo_l1b.py

```diff
@@ -241,3 +245,110 @@ def test_spin_cycle():
 
     # Assert
     np.testing.assert_array_equal(spin_cycle_data["spin_cycle"], spin_cycle_expected)
+
+
+def test_get_spin_start_times():
+    # Arrange
+    l1b_de = xr.Dataset(
+        {
+            "spin_cycle": ("epoch", [0, 1, 2, 3, 4]),
+        },
+        coords={
+            "epoch": [
+                0,
+                1,
+                2,
+                3,
+                4,
+            ]
+        },
+    )
+    l1a_de = xr.Dataset(
+        {
+            "de_count": ("epoch", [2, 3]),
+            "met": ("direct_event", [0, 1, 2, 3, 4]),
+            "de_time": ("direct_event", [0000, 1000, 2000, 3000, 4000]),
+        },
+        coords={"epoch": [0, 1], "direct_event": [0, 1, 2, 3, 4]},
+    )
+    spin = xr.Dataset(
+        {
+            "start_sec_spin": (
+                ["epoch", "spin"],
+                [[20, 25, 30, 35, 40], [45, 50, 55, 60, 65]],
+            ),
+            "start_subsec_spin": (
+                ["epoch", "spin"],
+                [[2000, 3000, 4000, 5000, 6000], [1000, 1500, 2000, 3000, 4000]],
+            ),
+        }
+    )
+
+    end_acq = xr.DataArray([0, 1], dims="epoch")
+    spin_start_times_expected = np.array([20.002, 50.0015, 55.002, 60.003, 65.004])
+    spin_start_times = get_spin_start_times(l1a_de, l1b_de, spin, end_acq)
+
+    np.testing.assert_allclose(
+        spin_start_times,
+        spin_start_times_expected,
+        atol=1e-4,
+        err_msg=f"Spin start times: {spin_start_times} vs\
+         expected spin start times {spin_start_times_expected}",
```

Comment:
This should automatically print these out for you with the numpy testing assertions and highlight the exact entries that were different so do you want to override that?

Reply:
Yeah, no reason to override that. I'll remove it